third_party/gfootball_engine/src/main.hpp

```diff
@@ -74,10 +74,20 @@ class GameConfig {
   int render_resolution_x = 1280;
   int render_resolution_y = 720;
   std::string updatePath(const std::string& path) {
+#ifdef WIN32
```

Comment:
Could we consider use same API (C++ 17 filesystem or use boost filesystem) instead split Win32 and Unix file path?